build/config.go

```diff
@@ -25,8 +28,9 @@ const (
 //
 //nolint:lll
 type LogConfig struct {
-	Console *consoleLoggerCfg `group:"console" namespace:"console" description:"The logger writing to stdout and stderr."`
-	File    *FileLoggerConfig `group:"file" namespace:"file" description:"The logger writing to LND's standard log file."`
+	Console   *consoleLoggerCfg `group:"console" namespace:"console" description:"The logger writing to stdout and stderr."`
+	File      *FileLoggerConfig `group:"file" namespace:"file" description:"The logger writing to LND's standard log file."`
+	BuildInfo string            `long:"build-info" description:"Whether or not to include build info in log lines." choice:"on" choice:"off"`
```

Comment:
I'm not sure if "build info" is super descriptive. How about a boolean that's just `no-git-commit` to turn off (but default on)?